portfolio/src/main/webapp/script.js

```diff
@@ -21,8 +21,23 @@ function addRandomGreeting() {
 
   // Pick a random greeting.
   const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
+    console.log("Current Greeting: " + greeting);
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact() {
+  const facts =
+      ['I am currently 20 years old!', 'I love to hike, bike, and camp!', 'I am an eagle scout！', 'I was the valedictorian of my high school!', 'I love traveling to other countries! I\'ve been to about 10.', 'I love talking to others and socializing with others!'];
+
+  // Pick a random fact.
+  const factNum = Math.floor(Math.random() * facts.length);
+  const fact = facts[factNum];
+  // Add it to the page.
+
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = fact;
+  const factNumContainer = document.getElementById('factNum-container');
+  factNumContainer.innerText = "This is fact number " + (factNum + 1) + "!";
```

Comment:
In general, be careful about magic constants that don't have great explanations. For example, I might consider doing something like:

const factNum = ...; // Randomize a constant to pick a fact to display
const factNumDisplayNumer -= factNum + 1; // Index is 0-based, but we want to display 1-based to user.

That way, you don't have a random "+ 1" somewhere in the code that isn't explicable.